members/STXLc4e68274.yaml

```diff
@@ -1,7 +1,7 @@
-bioguide: TXL000236
+bioguide: STXLc4e68274
 contact_form:
   method: post
-  action: /members/member-page/email/?district=8&session=84
+  action: ""
```

Comment:
Just FYI, this `action` is not used by any of our stuff (so you can always omit it if you want)

Reply:
👍

Reply:
and by omit I mean like leave the line out entirely